hackathon-topics.json

```diff
@@ -0,0 +1,10 @@
+[
+    {
+        "title": "Example Topic",
+        "description": "Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.",
+        "host": "Some Guy",
```

Comment:
Please use a gender neutral language as the rest of the orga team would like to have more women involved in the conference